odd-platform-ui/package.json

```diff
@@ -22,6 +22,11 @@
       "git add"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run tsc --noEmit && lint-staged"
+    }
+  },
```

Comment:
@Haarolean what do you think about that? It's necessary? Or can we take the "pre-commit" script from your PR?

Reply:
We should use mine, it takes into consideration changed files (if no frontend files have been changed, it does not get to run) and it doesn't run on CI ofc.

Reply:
done